mlir/test/Dialect/LLVMIR/indirectbr.mlir

```diff
@@ -0,0 +1,81 @@
+// RUN: mlir-opt -split-input-file --verify-roundtrip %s | FileCheck %s
+
+llvm.func @ib0(%dest : !llvm.ptr, %arg0 : i32, %arg1 : i32) -> i32 {
+  llvm.indirectbr %dest : !llvm.ptr, [
+  ^head(%arg0 : i32),
+  ^tail(%arg1 : i32)
```

Comment:
Can you maybe add two arguments in one of the successor operand lists. Are we printing the types after all the operands or do we mix operands and types?